docs/guides/administrator/adding-and-managing-tenants.md

```diff
@@ -0,0 +1,144 @@
+# Adding and Managing Tenants Guide
```

Comment:
Adding and Managing?

"Adding" is part of "managing" I suppose.

Maybe let's do "Adding and Removing Tenants" or just "Managing Tenants".